Update Vault's PKI External CA API docs (#3225)
hashicorp/web-unified-docs: content/vault/v2.x/content/api-docs/secret/pki-external-ca.mdx

  validating the TLS certificate provided by the ACME server. Use `trusted_ca`
  for private ACME servers with custom CAs.

- `default_nameserver` `(string: "")` - Address of a DNS nameserver (`host` or
  `host:port`) to append to the resolver list when verifying DNS-01 challenge
  propagation for DNS providers that do not specify a nameserver. When empty, no

>   extra resolver is appended beyond the authoritative primary NS. 

Reviewer: ```suggestion
  propagation for DNS providers that do not specify a nameserver. Leaving `default_nameserver` empty tells Vault not to append additional resolvers beyond the authoritative primary NS. 
```
Style correction: write in active voice

- `trusted_ca` `(string: "")` - PEM-encoded trusted CA certificates for the ACME server.

- `default_nameserver` `(string: "")` - Address of a DNS nameserver (`host` or
  `host:port`) to append to the resolver list when verifying DNS-01 challenge
  propagation for DNS providers that do not specify a nameserver. When empty, no

>   extra resolver is appended beyond the authoritative primary NS.

Reviewer: ```suggestion
  propagation for DNS providers that do not specify a nameserver. Leaving `default_nameserver` empty tells Vault not to append additional resolvers beyond the authoritative primary NS. 
```
Edited to match previous suggestion

- `ttl` `(duration: "60s")` - TTL for DNS TXT records used in DNS-01 challenges.

- `nameserver` `(string: "")` - DNS server address (`host` or `host:port`) to

>   use when verifying DNS-01 challenge propagation for this provider. When empty,

Reviewer: ```suggestion
  use when verifying DNS-01 challenge propagation for the configured provider. When empty,
```
Style correction: avoid "this" as a pronoun

  Vault falls back to the ACME account's `default_nameserver` (if set) and the

>   authoritative primary NS.

Reviewer: ```suggestion
  Vault falls back to the `default_nameserver` value from the ACME account (if set) and the authoritative primary NS.
```
Style correction: avoid possessives

- `ttl` `(duration: "60s")` - TTL for DNS TXT records used in DNS-01 challenges.

- `nameserver` `(string: "")` - DNS server address (`host` or `host:port`) to
  use when verifying DNS-01 challenge propagation for this provider. When empty,
  Vault falls back to the ACME account's `default_nameserver` (if set) and the

>   authoritative primary NS.

Reviewer: ```suggestion
  Vault falls back to the `default_nameserver` value from the ACME account (if set) and the authoritative primary NS.
```
Edited to match previous suggestion

- `ttl` `(duration: "10s")` - TTL for DNS TXT records used in DNS-01 challenges.

- `nameserver` `(string: "")` - DNS server address (`host` or `host:port`) to
  use when verifying DNS-01 challenge propagation for this provider. When empty,
  Vault falls back to the ACME account's `default_nameserver` (if set) and the

>   authoritative primary NS.

Reviewer: ```suggestion
  Vault falls back to the `default_nameserver` value from the ACME account (if set) and the authoritative primary NS.
```
Edited to match previous suggestion

- `identifiers` `(array<string>: <required>)` - List of DNS identifiers (domain names) the provider can manage. Supports wildcard patterns with leftmost asterisk (e.g., `*.example.com`).

> - `nameserver` `(string: <required>)` - DNS server address in `host` or `host:port` format (e.g., `192.168.1.1:53` or `192.168.1.1`). RFC2136 defaults to port 53 when no port is specified.

Reviewer: ```suggestion
- `nameserver` `(string: <required>)` - DNS server address in `host` or `host:port` format (e.g., `192.168.1.1:53` or `192.168.1.1`). RFC2136 defaults to port 53 when you do not specify a port.
```
Style correction: write in active voice
